modules/nf-core/sgdemux/main.nf

```diff
@@ -0,0 +1,49 @@
+process SGDEMUX {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::sgdemux=1.1.0"
+    if (workflow.profile.tokenize(',').intersect(['Singularity', 'Docker']).size() >= 1) {
+        exit 1, "SGDEMUX module does not support Singularity or Docker. Please use Conda / Mamba instead."
+    }
+    // TODO check sgdemux is not on singularity/docker (google says no) double check with Nils
+    // container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 
+        // 'https://depot.galaxyproject.org/singularity/YOUR-TOOL-HERE':
+        // 'quay.io/biocontainers/YOUR-TOOL-HERE' }"]
+
+    input:
+    tuple val(meta), path(run_manifest), path(run_dir)
```

Comment:
```suggestion
    tuple val(meta), path(sample_sheet), path(in_fastqs_dir)
```

Reply:
I'd also add a comment either here or above the process that the input FASTQs must be bgzipped!